backend/src/test/java/org/eclipse/tractusx/semantics/hub/ModelsApiFilterTest.java

```diff
@@ -112,7 +96,7 @@ public void testGetModelListByAvailablePropertyTypeExpectResultsFound() throws E
    public void testGetByCombinedFilters() throws Exception {
```

Comment:
You might be able to combine this test with the testGetByNamespaceExpectResultsFound (line 51), e.g., by extending the other method to a case where one does find a matching namespace. 

In any case, I would change the name of the test method because there a no combined filters tested anymore.